dasd driver changes:
 - Fix check for device type in error recovery for fba devices.

diffstat:
 drivers/s390/block/dasd.c          |    2 +-
 drivers/s390/block/dasd_3990_erp.c |    2 +-
 drivers/s390/block/dasd_cmb.c      |    2 +-
 drivers/s390/block/dasd_fba.c      |    4 ++--
 drivers/s390/block/dasd_genhd.c    |    2 +-
 5 files changed, 6 insertions(+), 6 deletions(-)

diff -urN linux-2.6/drivers/s390/block/dasd.c linux-2.6-s390/drivers/s390/block/dasd.c
--- linux-2.6/drivers/s390/block/dasd.c	Sun Apr  4 05:37:06 2004
+++ linux-2.6-s390/drivers/s390/block/dasd.c	Tue Apr 13 10:20:33 2004
@@ -7,7 +7,7 @@
  * Bugreports.to..: <Linux390@de.ibm.com>
  * (C) IBM Corporation, IBM Deutschland Entwicklung GmbH, 1999-2001
  *
- * $Revision: 1.136 $
+ * $Revision: 1.137.2.1 $
  */
 
 #include <linux/config.h>
diff -urN linux-2.6/drivers/s390/block/dasd_3990_erp.c linux-2.6-s390/drivers/s390/block/dasd_3990_erp.c
--- linux-2.6/drivers/s390/block/dasd_3990_erp.c	Sun Apr  4 05:38:00 2004
+++ linux-2.6-s390/drivers/s390/block/dasd_3990_erp.c	Tue Apr 13 10:20:33 2004
@@ -5,7 +5,7 @@
  * Bugreports.to..: <Linux390@de.ibm.com>
  * (C) IBM Corporation, IBM Deutschland Entwicklung GmbH, 2000, 2001
  *
- * $Revision: 1.28 $
+ * $Revision: 1.29.2.1 $
  */
 
 #include <linux/timer.h>
diff -urN linux-2.6/drivers/s390/block/dasd_cmb.c linux-2.6-s390/drivers/s390/block/dasd_cmb.c
--- linux-2.6/drivers/s390/block/dasd_cmb.c	Sun Apr  4 05:37:23 2004
+++ linux-2.6-s390/drivers/s390/block/dasd_cmb.c	Tue Apr 13 10:20:33 2004
@@ -1,5 +1,5 @@
 /*
- * linux/drivers/s390/block/dasd_cmb.c ($Revision: 1.6 $)
+ * linux/drivers/s390/block/dasd_cmb.c ($Revision: 1.7.2.1 $)
  *
  * Linux on zSeries Channel Measurement Facility support
  *  (dasd device driver interface)
diff -urN linux-2.6/drivers/s390/block/dasd_fba.c linux-2.6-s390/drivers/s390/block/dasd_fba.c
--- linux-2.6/drivers/s390/block/dasd_fba.c	Sun Apr  4 05:37:36 2004
+++ linux-2.6-s390/drivers/s390/block/dasd_fba.c	Tue Apr 13 10:20:33 2004
@@ -4,7 +4,7 @@
  * Bugreports.to..: <Linux390@de.ibm.com>
  * (C) IBM Corporation, IBM Deutschland Entwicklung GmbH, 1999,2000
  *
- * $Revision: 1.32 $
+ * $Revision: 1.33 $
  */
 
 #include <linux/config.h>
@@ -205,7 +205,7 @@
 		return dasd_era_none;
 	
 	cdev = device->cdev;
-	switch (cdev->id.dev_model) {
+	switch (cdev->id.dev_type) {
 	case 0x3370:
 		return dasd_3370_erp_examine(cqr, irb);
 	case 0x9336:
diff -urN linux-2.6/drivers/s390/block/dasd_genhd.c linux-2.6-s390/drivers/s390/block/dasd_genhd.c
--- linux-2.6/drivers/s390/block/dasd_genhd.c	Sun Apr  4 05:37:40 2004
+++ linux-2.6-s390/drivers/s390/block/dasd_genhd.c	Tue Apr 13 10:20:33 2004
@@ -9,7 +9,7 @@
  *
  * gendisk related functions for the dasd driver.
  *
- * $Revision: 1.46 $
+ * $Revision: 1.47.2.1 $
  */
 
 #include <linux/config.h>
